contributing.md: Clarify that all functional changes must be reviewed (#1089)
hyphanet/fred: CONTRIBUTING.md

people in [IRC](https://www.hyphanet.org/pages/help.html#chat-with-us) or the [mailing list](https://www.hyphanet.org/pages/help.html#mailing-lists) - we may
have missed it.

> All commits with functional changes must be reviewed, also those from

Reviewer: ```suggestion
All commits with functional changes must be reviewed, including those from
```

> maintainers. An exception are plugins: updates to official plugins

Reviewer: ```suggestion
maintainers. Updates to official plugins are an exception and
```

> must include the commit id from the the plugin-repo in the commit

Reviewer: ```suggestion
must include the commit ID from the the plugin repository in the commit
```

> description (the review in the plugin-repo doubles as review in fred).

Reviewer: ```suggestion
description; the review in the plugin repository then serves as review in fred.
```
